### ISSUE TITLE
Generate TypeScript typings from Flow

### ISSUE THREAD

**Issue by pleerock** [joarwilk/flowgen#36]
Would be great if this project will be able to generate typings from both directions - from ts to flow and versa from flow to ts.

**Comment by orta** [comment]
This project uses the TypeScript AST to generate a flow file, meaning it would require making that from scratch

I think there’s a flow to reason project that also does TS

**Comment by joarwilk** [comment]
Yeah the original intent of this project was to be a two-way swap but like @orta said that would essentially be two standalone projects, so I scrapped that idea.

**Comment by stereobooster** [comment]
There is [flow-to-typescript](https://github.com/bcherny/flow-to-typescript).  If you are looking for more conversion tools check [type-o-rama](https://github.com/stereobooster/type-o-rama).
